### PR TITLE
[NB] Add lights

### PR DESCRIPTION
**Pull request by LaStrada**

### CODE REVIEW

**Comment by svarthvitt** [comment]
@LaStrada I have been working on further expansions++ to give more flexibility, but I am a newbie and have not managed to branch them in myself.

  temperature:
    range:
      type: "temperature"
      from: 0
      to: 100
  temperature_unit:
    values:
      - "celsius"
      - in: "c"
        out: "celsius"
      - "fahrenheit"
      - in: "f"
        out: "fahrenheit"
expansion_rules:
  alle: "(alle | alt | samtlige)"
  farge: "{color}[t[t]]"
  i_på: "(i | på | ved [siden av])"
  lys: "(lys[(ene|et)] | lyskilde[r|n|ene] | [gløde|led|diode]lamp[e|en|er|ene] | lysrør[et|ene] | belysning[en])"
  lysstyrke: "{brightness}[%| prosent]"
  navn: "{name}[(en|et|n[s])]"
  område: "{area}[(n[s]|en|et[s]|met[s])]"
  skru_av: "(skru av | slukk | slå av | slukk | steng [av] | stopp)"
  skru_på: "(skru på | tenn | sett[e] (på | igang | fart på) | slå på | start[e])"
  hva: "(hva er | hva er det | hvilken | hvilke)"
  endre: "( sett | juster | skru (opp | ned) | vri | dimm[e] | still [inn] | øk | senk | dim[me])"
  temperatur: "{temperature}[°| grader] [{temperature_unit}]"
  åpne: "åpne [opp] | heis [opp] | lukk opp | rull opp"
  lukk: "lukk [igjen] | heis [ned] | rull [ned] | steng | senk"
  vindu: "[tak]vindu[et|er|ene] | [tak]luke[en|er|ene]"
  garasjedør: "garasj[edør|eport][(en|er|ene)]"
  gardin: "[rulle]gardin[(et|er|ene)] | persienn[et|er|ene] | forheng[et|er|ene]"
  persienne: "persienne[(n|r|ene)]"
  rullegardin: "rullegardin[(et|er|ene)]"
  skodde: "[rulle]skodde[(n|r|ene)] | solskjerm[(en|er|ene)]"
skip_words:
  - "vær så snill"
  - "takk"
  - "vennligts"
  - "kan du"

**Comment by LaStrada** [comment]
@svarthvitt I started with two small PRs (lights + temperature) so it was easier to review and to get some texts in. But feel free to open one (or multiple) PRs as well

**Comment by svarthvitt** [comment]
> @svarthvitt I started with two small PRs (lights + temperature) so it was easier to review and to get some texts in. But feel free to open one (or multiple) PRs as well
> 
> Edit: or if you want me to add them I can do so

Thank you. I have added https://github.com/home-assistant/intents/pull/904. Would you mind checking that I have not messed up anything in my attempt? I tried for a long time to send the PR from codespace, but it will not submit (access rights?).

File: sentences/nb/light_HassTurnOn.yaml [hunk 1]
```diff
@@ -2,7 +2,9 @@ language: 'nb'
 intents:
   HassTurnOn:
     data:
-      - sentences: []
+      - sentences:
```
Comment (svarthvitt) [inline sentences/nb/light_HassTurnOn.yaml:5]: Already merged in #904

File: tests/nb/light_HassTurnOn.yaml [hunk 2]
```diff
@@ -1,6 +1,69 @@
 language: 'nb'
 tests:
-- sentences: []
-  intent:
-    name: HassTurnOn
-    slots: {}
+  - sentences:
```
Comment (svarthvitt) [inline tests/nb/light_HassTurnOn.yaml:3]: Already merged in in #904. Not required.

File: sentences/nb/_common.yaml [hunk 3]
```diff
@@ -8,6 +8,11 @@ responses:
     no_entity: "Finner ingen enhet med navn {{ entity }}"
     handle_error: "En uventet feil oppsto"
 lists:
+  brightness:
```
Comment (svarthvitt) [inline sentences/nb/_common.yaml:11]: No changes to _common required. Your proposals are already merged in with #904. Note that they have changed requirements to no whitespaces around |, stated in new hassil  #830.

File: sentences/nb/light_HassLightSet.yaml [hunk 4]
```diff
@@ -2,4 +2,22 @@ language: nb
...
+          - "sett <område> [til] [lysstyrke[n]] [til] <lysstyrke>"
+        slots:
+          name: all
+        response: brightness
+      - sentences:
+          - "sett <navn> farge til <farge>"
+          - "sett [farge[n] <i_på>] <navn> til <farge>"
+        response: color
+      - sentences:
+          - "sett <farge>[t | tt] lys <i_på> <område>"
```
Comment (svarthvitt) [inline sentences/nb/light_HassLightSet.yaml:20]: We meed to remove whitespaces around |, stated in new hassil https://github.com/home-assistant/intents/pull/830.

**Review by svarthvitt — CHANGES_REQUESTED** [review]

File: tests/nb/light_HassTurnOn.yaml (hunk 2 above)
Comment (LaStrada) [reply tests/nb/light_HassTurnOn.yaml]: Thanks. But still a lot more to test, so will keep my changes.

**Review by svarthvitt — APPROVED** [review]
May I suggest you as the language leader for NB?

**Pull request merged**